omod/src/main/resources/config.xml

```diff
@@ -225,7 +225,7 @@
 
     <globalProperty>
         <property>coreapps.patientSearchHandler</property>
-        <defaultValue>patientByIdentifier</defaultValue>
+        <defaultValue></defaultValue>
         <description>
             Indicates which patient search handler implementation should be used by the patient search widget
```

Comment:
Update this to say something like "Set this to override the built-in search handlers with a custom search handler", or something like that?

Reply:
Done